src/odb/include/odb/db.h

```diff
@@ -1844,7 +1850,12 @@ class dbNet : public dbObject
   /// Returns driving term id assigned of this net. -1 if not set, 0 if non
   /// existent
   ///
-  int getDrivingITerm() const;
+  int getDrivingITermId() const;
```

Comment:
You can remove this API as the new getDrivingITerm makes more sense as a public API.